include/quic/endpoint.hpp

```diff
@@ -48,13 +48,13 @@ namespace oxen::quic
         void delete_connection(const ConnectionID& cid);
 
         Connection* get_conn(ConnectionID ID);
-        Connection* get_conn(Address addr);
+        Connection* get_conn(Address& addr);
```

Comment:
Since we aren't modifying this, it should be a `const Address&`.  Aside from const correctness, it has an important consequence for the caller: with a non-const lvalue reference the caller will be prohibited from passing a rvalue (i.e. temporary value) in, i.e. this would not be allowed:

```C++
ep.get_conn(Address{"1.2.3.4", 5678});
```

and so if you don't need to modify and just want to avoid copy, take arguments by const lvalue reference.